### PR TITLE
Markdown interface updates

### PR DESCRIPTION
**Pull request by benhaynes**

### CODE REVIEW

File: src/interfaces/markdown/input.vue [hunk 1]
```diff
@@ -1,14 +1,30 @@
...
+    <div
+      v-if="preview || !options.tabbed"
+      class="preview"
+      v-html="compiledMarkdown"
+      :style="{ height: options.rows * 23 + 'px' }"
+    ></div>
+    <div v-if="options.tabbed" class="toolbar">
+      <span @click="preview = false" :class="{ active: !preview }">
+        <v-icon name="code" size="22" />
+        Edit
```
Comment (rijkvanzanten) [inline src/interfaces/markdown/input.vue]: Lets use the translations for this, eg `$t('edit')`

File: src/interfaces/markdown/input.vue [hunk 2]
```diff
@@ -1,14 +1,30 @@
...
+      </span>
+      <span @click="preview = true" :class="{ active: preview }">
+        <v-icon name="visibility" size="22" />
+        Preview
```
Comment (rijkvanzanten) [inline src/interfaces/markdown/input.vue]: Use i18n

File: src/interfaces/markdown/input.vue [hunk 3]
```diff
@@ -19,6 +35,11 @@ import marked from "marked";
 import mixin from "@directus/extension-toolkit/mixins/interface";
 
 export default {
+  data() {
+    return {
+      preview: false
```
Comment (rijkvanzanten) [inline src/interfaces/markdown/input.vue]: Is this going the store the preview HTML or is it a flag that toggles between `previewMode` ?

Comment (benhaynes) [reply src/interfaces/markdown/input.vue]: Flag for when switching between edit and preview modes.

**Pull request merged**